bridge-fungible-token-abi/src/bridge_fungible_token_abi.sw

```diff
@@ -0,0 +1,38 @@
+library bridge_fungible_token_abi;
+
+use std::{contract_id::ContractId, identity::Identity, vm::evm::evm_address::EvmAddress};
+
+abi BridgeFungibleToken {
+    /// Claim a refund for an EvmAddress if one has been registered.
+    ///
+    /// # Arguments
+    ///
+    /// * `originator` - the EvmAddress that is entitled to a refund
+    /// * `asset` - the EvmAddress of the L1 token for the refund
+    #[storage(read, write)]
+    fn claim_refund(originator: b256, asset: b256);
+
+    /// Withdraw coins back to L1 and burn the corresponding amount of coins
+    /// on L2.
+    ///
+    /// # Arguments
+    ///
+    /// * `to` - the destination of the transfer (an Address or a ContractId)
+    ///
+    /// # Reverts
+    ///
+    /// * When no coins were sent with call
+    /// * When the wrong asset was sent with the call
+    #[storage(read)]
+    fn withdraw_to(to: b256);
+    /// Get the name of this token contract
+    fn name() -> str[8];
```

Comment:
I think we should expand the length of all these fixed length strings to 32. This is a pretty common size that Ethereum contracts try to stay under since there is no storage cost difference between 1 and 32 byte strings. Cost only goes up after hitting 33 bytes